avm/res/api-management/service/main.bicep

```diff
@@ -66,6 +66,7 @@ param sku string = 'Developer'
 
 @description('Optional. The instance size of this API Management service.')
 @allowed([
+  0
```

Comment:
Beware, this also requires a run of the `Set-AVMModule` script as this changes the template hash & the allowed set in the readme (:

Reply:
I am going to run it in my forked repo so I can use the Set-AVMModule script and re-do the PR. Thanks for the heads up!

Reply:
Hey @tony-box, I'm confused. Is @cvs79 your stage name, or how come this user wouldn't just run set script? 😄

Reply:
I thought I did but apparently it was not in my pr.

Reply:
All good. The readme is now correctly updated, but for some reason, the `bicep build` was skipped. The template hash should update from `4652975451954294914` to `7048259745072727177`

Reply:
Done. Probably some tooling issue on my end